packages/react-vapor/src/components/calendar/Calendar.tsx

```diff
@@ -163,6 +168,7 @@ export class Calendar extends React.Component<ICalendarProps, any> {
                 (calendarSelection.isRange && day.date.isSame(selectionStart, 'day')) || day.isLowerLimit;
             day.isUpperLimit = (calendarSelection.isRange && day.date.isSame(selectionEnd, 'day')) || day.isUpperLimit;
             day.color = isSelected ? calendarSelection.color : day.color;
+            day.isCountdown = !!this.props.countdown;
```

Comment:
if countdown is already a boolean, I guess you can drop the `!!`

Reply:
not if is optional, the linter shows an error

Reply:
Newbie question:
Is `!!this.props.countdown` return the actual value, or it only validate if this prop is undefined or not (for example, if countdown is false, does it returns false or true (because the value is defined)).

Reply:
```ts
day.isCountdown = this.props.countdown; // if countdown not set, day.isCountdown is undefined
day.isCountdown = !this.props.countdown; // if countdown not set but with single bang, day.isCountdown is true
day.isCountdown = !!this.props.countdown; // if countdown not set with double bang, day.isCountdown is false
day.isCountdown = !!this.props.countdown; // if countdown set as true, day.isCountdown is true
day.isCountdown = !!this.props.countdown; // if countdown set as false, day.isCountdown is false
```

Does that answer your question?

Reply:
Yes sir! Thanks!